outputs/all_stack_outputs.testoutput

```diff
@@ -0,0 +1,935 @@
+        status: ACTIVE
+        tenant_id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        updated: 'xxxxxxxxxxxxxxxxxxxx'
+        user_id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        volumes: []
+    server:
+        HUMAN_ID: true
+        NAME_ATTR: name
+        OS-DCF:diskConfig: MANUAL
+        OS-EXT-AZ:availability_zone: nova
+        OS-EXT-STS:power_state: 1
+        OS-EXT-STS:task_state: null
+        OS-EXT-STS:vm_state: active
+        OS-SRV-USG:launched_at: 'xxxxxxxxxxxxxxxxxxxxxxxxxx'
+        OS-SRV-USG:terminated_at: null
+        accessIPv4: xxxxxxxxxxxx
+        accessIPv6: ''
+        addresses:
+            e2e-openstack:
+            -   OS-EXT-IPS-MAC:mac_addr: xxxxxxxxxxxxxxxxx
+                OS-EXT-IPS:type: fixed
+                addr: xxxxxxxxxxxxxx
+                version: 4
+            -   OS-EXT-IPS-MAC:mac_addr: xxxxxxxxxxxxxxxxx
+                OS-EXT-IPS:type: floating
+                addr: xx.x.xxx.xxx
+                version: 4
+        adminPass: xxxxxxxxxxxx
+        az: nova
+        cloud: defaults
+        config_drive: ''
+        created: 'xxxxxxxxxxxxxxxxxxxx'
+        flavor:
+            id: '2'
+            name: m1.small
+        hostId: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        human_id: testgroup4_ano_inst_0
+        id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        image:
+            id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+            name: rhel-6.5_jeos
+        interface_ip: xxxxxxxxxxxx
+        key_name: ci-factory
+        metadata: {}
+        name: xxxxxxxxxxxxxxxxxxxx
+        networks:
+            e2e-openstack:
+            - xxxxxxxxxxxxxx
+            - xxxxxxxxxxxx
+        os-extended-volumes:volumes_attached: []
+        private_v4: xxxxxxxxxxxxxx
+        progress: 0
+        public_v4: xxxxxxxxxxxx
+        public_v6: ''
+        region: ''
+        security_groups:
+        -   description: Default security group
+            id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+            name: default
+            security_group_rules:
+            -   direction: ingress
+                ethertype: IPv4
+                id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+                port_range_max: null
+                port_range_min: null
+                protocol: null
+                remote_ip_prefix: null
+                security_group_id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+            -   direction: ingress
+                ethertype: IPv4
+                id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+                port_range_max: null
+                port_range_min: null
+                protocol: null
+                remote_ip_prefix: null
+                security_group_id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        status: ACTIVE
+        tenant_id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        updated: 'xxxxxxxxxxxxxxxxxxxx'
+        user_id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        volumes: []
+os_volume_res: []
+rax_server_res: []
```

Comment:
I wonder if all of this data could be mocked using https://docs.python.org/3/library/unittest.mock.html. Thoughts here?